packages/patternfly-4/react-charts/src/components/ChartArea/examples/chart-area.scss

```diff
@@ -0,0 +1,14 @@
+.ws-preview {
+  & > * {
+    .chart-overflow {
+      & svg {
+        overflow: visible;
+      }
+    }
+
+    .chart-title {
+      margin-bottom: 25px;
```

Comment:
Should we be using SASS variables for things that are common in overflow, title, container, margin, etc?

Reply:
this was just ported from previous examples and deleted, we can refactor later if you'd like

Reply:
Okay we can do it later thought this was something new.  Thanks.